packages/eslint-mdx/src/helpers.ts

```diff
@@ -181,6 +141,82 @@ export const requirePkg = async <T>(
   throw error
 }
 
+/* istanbul ignore next -- used in worker */
+export const getPositionAt = (text: string, offset: number): Position => {
```

Comment:
[`vfile-location`](https://github.com/vfile/vfile-location) might be useful here?

Reply:
Maybe in next development schedule. I'm a bit tied today. 😂

This package is ESM only, to use it I need to do some refactor.

Reply:
You could use the previous version, but also fine without!